### PR TITLE
Module & Imports conformed to PEP8

### PR DESCRIPTION
**Pull request by FukurouMakoto**
To handle [Issue #779 ](https://github.com/mozilla/OpenWPM/issues/779).

Modules and their imports have been changed to snake_case.

### CODE REVIEW

**Comment by FukurouMakoto** [comment]
Could use some guidance on why these last two checks are failing; they don't seem to be due to a improper imports from what I'm reading in the error logs.

**Comment by vringar** [comment]
I'm surprised to see these tests failing seeing as they are working on [our current master branch](https://travis-ci.org/github/mozilla/OpenWPM/builds/745242480). Can you merge the current master into your branch and see if the tests keep failing?

**Comment by FukurouMakoto** [comment]
> merge the current master into your branch

Will do but would you mind telling me how to do so?
I've tried merging and as well as rebase and git is saying my branch is up to date.

**Comment by ankushduacodes** [comment]
@vringar I just looked at @FukurouMakoto's master branch, It's up-to-date. I tried running the failed tests locally, they are running fine.
@FukurouMakoto Try running the failed tests locally may be with this command `python -m pytest -vv -s -k test_parse_neterror_integration` just change test name every time `python -m pytest -vv -s -k <TEST_NAME>`

Dont forget to do `cd test` in project directory

**Comment by vringar** [comment]
> Will do but would you mind telling me how to do so?
> I've tried merging and as well as rebase and git is saying my branch is up to date.

Sorry for not checking that first. I was hoping this could fix these issues.

File: OpenWPM.code-workspace [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "folders": [
+        {
+            "path": "openwpm"
+        }
+    ]
+}
```
Comment (vringar) [inline OpenWPM.code-workspace:7]: Could you please remove this file and add it to the .gitignore explaining what it is

File: openwpm/Commands/browser_commands.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
 from selenium.webdriver.support import expected_conditions as EC
 from selenium.webdriver.support.ui import WebDriverWait
 
-from ..SocketInterface import clientsocket
+from ..socket_interface import clientsocket
```
Comment (vringar) [inline openwpm/Commands/browser_commands.py:23]: Could you rename `clientsocket` to `ClientSocket`

File: test/test_webdriver_utils.py [hunk 3]
```diff
@@ -22,13 +22,13 @@ def get_config(self, data_dir=""):
...
         manager_params, browser_params = self.get_config()
-        manager = TaskManager.TaskManager(manager_params, browser_params)
+        manager = task_manager.TaskManager(manager_params, browser_params)
         manager.get("http://website.invalid")
         manager.close()
 
         get_command = db_utils.query_db(
             manager_params["db"],
-            "SELECT command_status, error FROM crawl_history WHERE command = \"<class 'openwpm.Commands.Types.GetCommand'>\"",
+            "SELECT command_status, error FROM crawl_history WHERE command = \"<class 'openwpm.Commands.types.GetCommand'>\"",
```
Comment (vringar) [inline test/test_webdriver_utils.py:31]: Could you please change `Commands` to `commands`.

File: openwpm/DeployBrowsers/deploy_firefox.py [hunk 4]
```diff
@@ -9,7 +9,7 @@
 from selenium.webdriver.firefox.firefox_profile import FirefoxProfile
 
 from ..Commands.profile_commands import load_profile
```
Comment (vringar) [inline openwpm/DeployBrowsers/deploy_firefox.py:11]: Please change `DeployBrowsers` to `deploy_browsers`

**Review by vringar — CHANGES_REQUESTED** [review]
Thank you for all the work you put into this!
Would you mind working on this some more and renaming the things I commented?

File: docs/Using_OpenWPM.md [hunk 5]
```diff
@@ -20,7 +20,7 @@ Suppose we want to add a top-level command to cause the browser to jiggle the mo
 
 To add a new command you need to modify the following four files:
 
-1. Define all required paramters in a type in `openwpm/Commands/Types.py`  
+1. Define all required paramters in a type in `openwpm/commands/Types.py`
```
Comment (vringar) [inline docs/Using_OpenWPM.md:23]: This should be 
```suggestion
1. Define all required paramters in a type in `openwpm/commands/types.py`  
```
right?

File: openwpm/mp_logger.py [hunk 6]
```diff
@@ -16,7 +16,7 @@
 from sentry_sdk.integrations.logging import BreadcrumbHandler, EventHandler
 from tblib import pickling_support
 
-from .Commands.utils.webdriver_utils import parse_neterror
+from .commands.utils.webdriver_utils import parse_neterror
 from .socket_interface import serversocket
```
Comment (vringar) [inline openwpm/mp_logger.py:20]: Can you please turn `serversocket` into `ServerSocket`. Sorry for missing this before.

**Review by vringar — CHANGES_REQUESTED** [review]

**Review by vringar — APPROVED** [review]

**Pull request merged**